packages/edit-site/src/components/dataviews/dataviews.js

```diff
@@ -14,6 +14,22 @@ import Pagination from './pagination';
 import ViewActions from './view-actions';
 import TextFilter from './text-filter';
 import { ViewGrid } from './view-grid';
+import { ViewSideBySide } from './view-side-by-side';
+
+// To do: convert to view type registry.
+export const viewTypeSupportsMap = {
+	list: {},
```

Comment:
I feel we should rename this one "table" and the new one "list"

Reply:
I don't understand what you mean here. You named it list and grid?

Reply:
I think that makes some sense
* rename "list" to "table": totally on board
* rename "side-by-side" to "list": not as clear, but functionally the left-hand side is a list